common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -6,9 +6,8 @@
 
 import json
 import logging
-import warnings
-
 from lxml import etree
+import warnings
```

Comment:
`lxml` is a third-party package, so it belongs in a second group of imports with imports from the `xblock` package. I noticed `pkg_resources` below; it should go with the other standard library imports.